packages/console/src/pages/TenantSettings/TenantBasicSettings/ProfileForm/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { useContext } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+
+import CopyToClipboard from '@/components/CopyToClipboard';
+import FormCard from '@/components/FormCard';
+import FormField from '@/components/FormField';
+import RadioGroup, { Radio } from '@/components/RadioGroup';
+import TextInput from '@/components/TextInput';
+import { TenantsContext } from '@/contexts/TenantsProvider';
+
+import { type TenantSettingsForm } from '../types.js';
+
+import * as styles from './index.module.scss';
+import { tagOptions } from './options.js';
+
+function ProfileForm() {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const {
+    control,
+    register,
+    formState: { errors },
+  } = useFormContext<TenantSettingsForm>();
+  const { currentTenantId } = useContext(TenantsContext);
+
+  return (
+    <FormCard title="tenant_settings.profile.title">
+      <FormField title="tenant_settings.profile.tenant_id">
+        <CopyToClipboard value={currentTenantId} variant="border" className={styles.textField} />
+      </FormField>
+      <FormField isRequired title="tenant_settings.profile.tenant_name">
+        <TextInput
+          {...register('profile.name', { required: true })}
+          error={Boolean(errors.profile?.name)}
+        />
+      </FormField>
+      <FormField title="tenant_settings.profile.environment_tag">
+        <Controller
+          control={control}
+          name="profile.tag"
+          rules={{ required: true }}
```

Comment:
This field has default value, so need to be `required`

Reply:
removed